pkg/cmd/utility/utility.go

```diff
@@ -10,7 +10,6 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 )
 
```

Comment:
I think we should remove this function and its usages/references.  My related PR will add new messages.

`func DisplayUnsealedSecretsWarning() {`